src/App.css

```diff
@@ -242,3 +243,215 @@ form h3 {
   padding: 100px 100px 150px;
   width: 100%;
 }
+
+@media (max-width: 768px) {
+  nav {
+    width: 100%;
+  }
+
+  .nav-menu {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav-menu p {
+    font-size: 0.875rem;
+    color: #0290ff;
+    font-weight: bold;
+  }
+
+  .nav-menu ul {
+    list-style: none;
+    display: flex;
+    gap: 20px;
+    margin-left: 20px;
+  }
+
+  .nav-menu ul li a {
+    font-size: 0.413rem;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 1.9px;
+    color: #121212;
+  }
+
+  .userIconCont {
+    display: none;
+  }
+
+  .booksContainer {
+    padding: 50px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  form h3 {
+    font-size: 1rem;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: -0.18px;
+    color: #888;
+  }
+
+  .form-control {
```

Comment:
- [ ] The form for adding a new book should have the same width as the book list, kindly increase the width of the form so it looks like the original design

![Screenshot_20220922_184127](https://user-images.githubusercontent.com/25227022/191815916-f565fb23-6d42-4eea-8aea-52b4b12787f9.png)

Reply:
Done